### PR TITLE
Build with Go 1.16

### PR DESCRIPTION
**Pull request by yselkowitz**
Closes #379

### CODE REVIEW

**Comment by timflannagan** [comment]
We may need to also update the go.mod module to 1.16 and re-generate the vendor tree.

/ok-to-test
/lgtm

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.6 AS builder
```
Comment (timflannagan) [inline Dockerfile.rhel7:2]: Ah, yeah this needs to be `registry.ci...`.

```suggestion
FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.6 AS builder
```

**Comment by timflannagan** [comment]
```
            			            	Operation cannot be fulfilled on operatorhubs.config.openshift.io "cluster": the object has been modified; please apply your changes to the latest version and try again
```

/retest
/lgtm

**Comment by yselkowitz** [comment]
getting closer...
/retest

**Pull request merged**